modules/dashboard/sections/resources/main.tf

```diff
@@ -104,6 +142,14 @@ locals {
       height = local.unit,
       width  = local.unit,
       widget = module.received_bytes.widget,
+    },
+    {
+      yPos   = local.unit * 2,
```

Comment:
shift others down, and make this the top row with `module.width.size` ?

Reply:
fixed.